### ISSUE TITLE
Project mission

### ISSUE THREAD

**Issue by tawfeeka** [amosproj/amos2023ss02-open-search-meta-data-hub#20]
## User story
1. As a team
2.  I need a project mission statement that clearly defines the problem we aim to solve or the goal we wish to achieve.
3. So that every team member can align their work accordingly.

## Acceptance criteria
* This should be well documented in the planning document.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by motrell** [comment]
Have read through it, sounds good!
